addons/mail/static/src/models/composer/composer.js

```diff
@@ -473,10 +588,32 @@ function factory(dependencies) {
             }
         }
 
+        _getSuggestedCannedResponses(mentionKeyword) {
+            this.update({
+                suggestedCannedResponses: [[
+                    'replace',
+                    this.env.messaging.cannedResponses.filter(
+                        cannedResponse => cannedResponse.source.includes(mentionKeyword)
+                    )
+                ]],
+            });
+
+            if (this.suggestedCannedResponses[0]) {
+                this.update({
+                    activeSuggestedCannedResponse: [['link', this.suggestedCannedResponses[0]]],
+                });
+            } else {
+                this.update({
+                    activeSuggestedCannedResponse: [['unlink']],
+                });
+            }
+        }
+
         /**
          * @private
          */
         _reset() {
+            this.closeCannedResponseSuggestions();
```

Comment:
`closeCannedResponseSuggestions` & `closeMentionSuggestions`. Latter should clarify partner mentions, but the anomaly is that both suggestions are treated too separately.